packages/auth/src/accessToken/routes.test.ts

```diff
@@ -207,4 +208,136 @@ describe('Access Token Routes', (): void => {
       })
     })
   })
+
+  describe('Revocation', (): void => {
+    let grant: Grant
+    let token: AccessToken
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    let tokenQuery: <M extends { QueryBuilderType: any }>(
+      trxOrKnex?: Objection.TransactionOrKnex | undefined
+    ) => Objection.QueryBuilderType<M>
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    let actualTokenQuery: <M extends { QueryBuilderType: any }>(
+      trxOrKnex?: Objection.TransactionOrKnex | undefined
+    ) => Objection.QueryBuilderType<M>
+
+    beforeAll(
+      async (): Promise<void> => {
+        actualTokenQuery = AccessToken.query
+      }
+    )
+
+    beforeEach(
+      async (): Promise<void> => {
+        grant = await Grant.query(trx).insertAndFetch({
+          ...BASE_GRANT
+        })
+        token = await AccessToken.query(trx).insertAndFetch({
+          grantId: grant.id,
+          ...BASE_TOKEN
+        })
+        tokenQuery = AccessToken.query
+      }
+    )
+
+    afterEach(
+      async (): Promise<void> => {
+        AccessToken.query = actualTokenQuery
+      }
+    )
+
+    test('Returns status 404 if token does not exist', async (): Promise<void> => {
+      const ctx = createContext(
+        {
+          headers: { Accept: 'application/json' }
+        },
+        {}
```

Comment:
The path parameter `id` is missing here.

Reply:
@dclipp The empty object in the `createContext` call _is_ the params object, so the `id` could be moved there. Does including the `params` in the first argument get it passed into the context?

See https://github.com/interledger/rafiki/blob/236912d7d12ed012d5e7f51496879ccc4f8901d8/packages/auth/src/tests/context.ts#L8 and https://github.com/interledger/rafiki/blob/236912d7d12ed012d5e7f51496879ccc4f8901d8/packages/auth/src/tests/context.ts#L14

Reply:
Good catch, I moved it to the empty object and was able to eliminate the mock for the `404` test